content/accomplishments.md

```diff
@@ -3,10 +3,17 @@ title: Accomplishments
 layout: 'infopage'
 ---
 
-[CTFtime Profile](https://ctftime.org/team/12858)
+
+## Various Acomplishments 
```

Comment:
i have very conflicted feelings about this year's ctf so i'd rather just include that we won it down below and remove this section entirely. i like the moving of the ctftime link underneath the header.

i will update you once i have the public bsidespdx ctf repo made, so can you put in an issue about replacing that link once we merge this PR?